recruiter/tests.py

```diff
@@ -14,9 +16,10 @@ def example_user():
 
 @pytest.fixture
 def example_recruiter(example_user):
-    recruiter = Recruiter(user=example_user, name="yarin bouzaglo",
-                          company=Company.objects.get(name="example_company_a"), email="test@yarin@example.com",
-                          phone_number="0524434795")
+    recruiter = Recruiter(user=example_user, name="test_recruiter",
+                          company=Company.objects.get(name="example_company_a"), email="test@email",
+                          phone_number="0543534343")
+
     recruiter.save()
```

Comment:
since it's a fixture and you only need it for the tests, it doesn't seem right to me that it's saved into the DB. @ifireball thoughts?

Reply:
When we just created the models we used pytest fixtures as well,  and of what I remember each test creates some sort of a temporary data base so the instance isn't saved in the real data base.